app/src/test/java/org/mozilla/fenix/components/metrics/MetricControllerTest.kt

```diff
@@ -583,7 +583,7 @@ class MetricControllerTest {
             CreditCardAutofillDialogFacts.Items.AUTOFILL_CREDIT_CARD_PROMPT_EXPANDED to CreditCards.autofillPromptExpanded,
             CreditCardAutofillDialogFacts.Items.AUTOFILL_CREDIT_CARD_PROMPT_DISMISSED to CreditCards.autofillPromptDismissed,
             CreditCardAutofillDialogFacts.Items.AUTOFILL_CREDIT_CARD_CREATED to CreditCards.savePromptCreate,
-            CreditCardAutofillDialogFacts.Items.AUTOFILL_CREDIT_CARD_UPDATED to CreditCards.savePromptUpdate,
```

Comment:
Did we want to remove this item event from being tested?

Reply:
It was not intentional. Fixed it now.